src/middlewared/middlewared/plugins/pool_/dataset_details.py

```diff
@@ -325,19 +325,7 @@ def build_details(self, mntinfo):
 
             results['vm'].append(vm)
 
-        # app
-        for app_name, paths in self.middleware.call_sync('chart.release.get_consumed_host_paths').items():
-            # We want to filter out any other paths which might be consumed to improve performance here
-            # and avoid unnecessary mount info calls i.e /proc /sys /etc/ etc
-            for path in filter(
-                lambda x: x.startswith('/mnt/') and 'ix-applications/' not in x,
-                paths
-            ):
-                results['app'].append({
-                    'name': app_name,
-                    'path': path,
-                    'mount_info': self.get_mount_info(path, mntinfo),
-                })
+        # FIXME: Add app details here, currently it would be an empty list
```

Comment:
Let's still return an "apps" key with an empty list so it doesn't break UI on the datasets page.

Reply:
We are doing that, it's already defined as empty above

Reply:
@yocalebo here https://github.com/truenas/middleware/blob/fd8e3e12ff05adf5b4d7743696e9ee8a22bd8fa3/src/middlewared/middlewared/plugins/pool_/dataset_details.py#L271-L271